src/kibali/AuthZChecker.cs

```diff
@@ -267,7 +267,8 @@ private static string CleanRequestUrl(string requestUrl)
                 return requestUrl;
             }
 
-            var parensRemoved = Regex.Replace(requestUrl.ToLowerInvariant(), @"\/\(.*?\)", string.Empty, RegexOptions.None, TimeSpan.FromSeconds(5)).Replace(@"//", "/");
+            var quotesRemoved = requestUrl.Replace("'", string.Empty);
+            var parensRemoved = Regex.Replace(quotesRemoved.ToLowerInvariant(), @"\/\(.*?\)", string.Empty, RegexOptions.None, TimeSpan.FromSeconds(5)).Replace(@"//", "/");
```

Comment:
We should probably also make this static Regex instance with the compiled flag passed in as well. 

https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-regex#static-regular-expressions